docs/source/users/index.md

```diff
@@ -230,6 +230,37 @@ response.  In this example, the endpoint returns an object with the schema
 `{"generated_texts":["<output>"]}`, hence the response path is
 `generated_texts.[0]`.
 
+### GPT4All usage (early-stage)
+
+Currently, we offer experimental support for GPT4All. To get started, first
+decide which models you will use. We currently offer three models from GPT4All:
+
+| Model name                   | Model size | Model bin URL                                              |
+|------------------------------|------------|------------------------------------------------------------|
+| `ggml-gpt4all-l13b-snoozy`   | 7.6 GB     | `http://gpt4all.io/models/ggml-gpt4all-l13b-snoozy.bin`    |
+| `ggml-gpt4all-j-v1.2-jazzy`  | 3.8 GB     | `https://gpt4all.io/models/ggml-gpt4all-j-v1.2-jazzy.bin`  |
+| `ggml-gpt4all-j-v1.3-groovy` | 3.8 GB     | `https://gpt4all.io/models/ggml-gpt4all-j-v1.3-groovy.bin` |
+
+
+Note that each model comes with its own license, and that users are themselves
+responsible for verifying that their usage complies with the license. You can
+find licensing details on the [GPT4All official site](https://gpt4all.io/index.html).
+
+For each model you use, you will have to run the command
+
+```
+curl -LO --output-dir ~/.cache/gpt4all "<model-bin-url>"
```

Comment:
What is the correct path for the cache on windows-machines running a local Jupyterlab?